gsmphys/sfc_noahmp_drv.f

```diff
@@ -819,6 +819,11 @@ subroutine noahmpdrv                                              &
           tem     = 1.0 / rho(i)
           hflx(i) = hflx(i) * tem * cpinv
           evap(i) = evap(i) * tem * hvapi
+          if (evap(i) .lt. -maxevap(i)) then
+            chh(i) = -maxevap(i)/evap(i)*chh(i)
+            hflx(i) = -maxevap(i)/evap(i)*hflx(i)
```

Comment:
Checking order of operations is supposed to be `(maxevap/evap) * _var_` and you want to negate it.

Reply:
There were two questions here.  One was to make sure it wasn't supposed to be -maxevap/(evap * _var_) and secondly questioning the use of the negative sign.